Sources/TuistKit/Extensions/ServiceContext+Tuist.swift

```diff
@@ -22,6 +22,8 @@ struct IgnoreOutputPipeline: StandardPipelining {
 
 extension ServiceContext {
     public static func tuist(_ action: (Path.AbsolutePath) async throws -> Void) async throws {
+        try setupEnv()
```

Comment:
Not sure if this was intentional or not, but the implementation of `setupEnv` was extracted from this method into `setupEnv()` in #7532, but it was never called. 

I think this would have stopped `--verbose` from working, although that _might_ have been intentional? 

I've added the call back, but maybe I need to remove the verbose checks again?

Reply:
> I think this would have stopped --verbose from working, although that might have been intentional?

Not intentional at all. This is a regression.

Reply:
I extracted the fix into [this PR](https://github.com/tuist/tuist/pull/7573)